az-core/src/main/java/azkaban/Constants.java

```diff
@@ -480,4 +481,15 @@
     // common private properties for multiple plugins
     public static final String COMMONSYSCONFFILE = "commonprivate.properties";
   }
+
+  public static class ContainerizedExecutionManagerProperties {
+    public static final String CONTAINERIZED_IMPL_TYPE = "azkaban.containerized.impl"
```

Comment:
azkaban.containerized prefix can be refactored to be a constant for a better type safety

Reply:
Taken care of. Thanks.